code/game/objects/items/control_wand.dm

```diff
@@ -29,6 +29,13 @@
 	to_chat(user, "Now in mode: [mode].")
 
 /obj/item/weapon/door_remote/afterattack(obj/machinery/door/airlock/D, mob/user)
+	if(istype(D)) //yes, the istype has to be here, otherwise it runtimes if you misclick
+		if(D.is_special)
+			if(!(D.check_access(src.ID)))
```

Comment:
It is not necessary, I just wanted it to display the access message instead
of the special message if you didn't have access. I will strip it out.

On Tue, May 17, 2016, 22:38 Fox McCloud <notifications@github.com> wrote:

> In code/game/objects/items/control_wand.dm
> <https://github.com/ParadiseSS13/Paradise/pull/4373#discussion_r63599129>:
>
> > @@ -29,6 +29,13 @@
> >  	to_chat(user, "Now in mode: [mode].")
> >
> >  /obj/item/weapon/door_remote/afterattack(obj/machinery/
>
> door/airlock/D, mob/user)
> > +	if(istype(D)) //yes, the istype has to be here, otherwise it runtimes if you misclick
> > +		if(D.is_special)
> > +			if(!(D.check_access(src.ID)))
>
> I don't understand why this is all necessary. If the door is special and
> doesn't work with the remote, why do you need to do an access check with it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ParadiseSS13/Paradise/pull/4373/files/50042c94c5541949680172452bb6fcfc25a4258e#r63599129>
>